.github/workflows/go.yml

```diff
@@ -25,18 +25,18 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        go: [ '1.20', '1.21', '1.22' ]
+        go: [ '1.21', '1.22' ]
         os: [ubuntu-latest, windows-2022, macos-14]
         exclude:
           # Only latest Go version for Windows and MacOS.
-          - os: windows-2022
-            go: '1.20'
           - os: windows-2022
             go: '1.21'
-          - os: macos-14
-            go: '1.20'
+          - os: windows-2022
+            go: '1.22'
```

Comment:
technically it is not correct for this commit since you have excluded every version for windows and mac here